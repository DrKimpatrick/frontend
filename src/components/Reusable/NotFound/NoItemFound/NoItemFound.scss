.noItemFound {
  position: relative;
  margin-top: 2rem;
  display: flex;
  align-items: center;
  .notItemImage {
    width: 80px;
    height: 80px;
    img {
      width: 100%;
      height: 100%;
      object-fit: contain;
    }
  }
  .message {
    font-weight: bold;
    font-size: 15px;
    margin-left: 1rem;
  }
}
