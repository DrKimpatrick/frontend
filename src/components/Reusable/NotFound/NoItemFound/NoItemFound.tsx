import React from 'react';
import './NoItemFound.scss';
import Image from 'assets/images/itemNotFound.png';

interface Props {
  message?: string;
}

const NoItemFound = (props: Props) => {
  const { message } = props;
  return (
    <div className="noItemFound">
      <div className="notItemImage">
        <img src={Image} alt="" />
      </div>
      <div className="message">{message || 'No result found'}</div>
    </div>
  );
};

export default NoItemFound;
