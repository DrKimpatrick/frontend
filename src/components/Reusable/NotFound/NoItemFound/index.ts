import NoItemFound from './NoItemFound';

export { NoItemFound };
