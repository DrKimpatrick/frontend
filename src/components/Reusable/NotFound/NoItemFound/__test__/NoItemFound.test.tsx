import React from 'react';
import ReactDom from 'react-dom';
import NoItemFound from '../NoItemFound';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';

describe('NoItemFound', () => {
  afterEach(cleanup);
  it('Should render without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(<NoItemFound message={'no result found'} />, div);
  });

  it('should create snapshot', () => {
    const tree = renderer
      .create(<NoItemFound message={'no result found'} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
