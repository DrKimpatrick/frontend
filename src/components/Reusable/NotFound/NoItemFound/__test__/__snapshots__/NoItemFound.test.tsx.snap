// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItemFound should create snapshot 1`] = `
<div
  className="noItemFound"
>
  <div
    className="notItemImage"
  >
    <img
      alt=""
      src="itemNotFound.png"
    />
  </div>
  <div
    className="message"
  >
    no result found
  </div>
</div>
`;
